src/features/deposit/components/Form/DepositFormNear/index.tsx

```diff
@@ -24,15 +24,15 @@ export type DepositFormNearValues = {
 
 export interface DepositFormNearProps {
   asset: BaseAssetInfo
-  signAndSendTransactionsNear: (transactions: Transaction[]) => void
+  sendTransaction: (transactions: Transaction[]) => void
```

Comment:
We should be explicit here that we send "Near" transaction. Say we need to implement sending funds directly from the widget, then most likely we will have a bunch of methods like `sendTransactionNear` or `sendTransactionSolana`.

Also it sendTransaction**s** because it accept list of transactions.

And return type might be also incorrect.

Reply:
Return type isn't finished yet.

Reply:
This is the DepositFormNear, so we are already in the NEAR context, there’s no need to be more implicit

Reply:
As for me having different names for one thing within a single feature increases congnitive weight.